CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## Unreleased
 ### Fixed
 - Fixed race condition during tests parallel execution (`tests::ingest::test_fetch::*` test group)
+- Fixed missed `flush()`'s for `tokio::fs:file`'s
```

Comment:
Note that changelog is not for us - it's for users of our app / libraries.

So the above line is something internal that I wouldn't mention at all.

This line about flush could be explained better from user perspective, e.g.:
```
Fixed async file flushing issues that could result in a race condition when using containerized ingest.
```